examples/examples_go_test.go

```diff
@@ -5,17 +5,32 @@
 package examples
 
 import (
+	"fmt"
+	"os"
 	"path/filepath"
 	"testing"
 
 	"github.com/pulumi/pulumi/pkg/v3/testing/integration"
+	"github.com/stretchr/testify/require"
 )
 
 func getGoBaseOptions(t *testing.T) integration.ProgramTestOptions {
+	goDepRoot := os.Getenv("PULUMI_GO_DEP_ROOT")
```

Comment:
How is this change related to the v6 upgrade (other than bumping to v6)?

Reply:
This was just needed to be able to run the tests locally in order to fix them up without going round via CI.